.github/workflows/release.yaml

```diff
@@ -90,9 +90,7 @@ jobs:
 
   publish:
     name: Publish Packages
-    needs: build
-    environment:
-      name: release
+    needs: release
```

Comment:
We should still keep the env as release right? That way the maintainers can approve the env and only the publish the package. Or are we removing that functionality, and going directly to release if the test and builds pass?

Reply:
I didn't think that was necessary since we'd already have to approve the environment on the release job. What do you think?

Reply:
oh, yes. I missed that in the diff. Apologies, this makes sense then 👍